packages/ds-medicare-gov/src/styles/components/_override.dialog.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable declaration-no-important -- temporarily ignore until mgov can be refactored */
```

Comment:
It's unknown what mgov is overwriting and why - should be migrated up the chain for tokens? Need a ticket for deeper investigation